src/StyledFormComponents.js

```diff
@@ -21,7 +21,7 @@ const StyledCard = styled.div`
   border-color: black;
   border-radius: 25px;
   background: #dbd8db;
-  width: 65%;
+  width: 900px;
```

Comment:
Whats the reasoning behind this?

Reply:
We did this as a fix for an issue we were having with the text and buttons overflowing out of the card when it was resized.